python/test/unit/language/test_matmul.py

```diff
@@ -468,7 +468,8 @@ def block_scale_mxfp_matmul(  #
                                                        (128, 128, 256), (128, 256, 256)])
 @pytest.mark.parametrize("NUM_STAGES", [1, 2, 4])
 @pytest.mark.parametrize("USE_2D_SCALE_LOAD", [False, True])
-@pytest.mark.skipif(torch.cuda.get_device_capability()[0] < 10, reason="Requires compute capability >= 10")
+@pytest.mark.skipif(not is_cuda() or torch.cuda.get_device_capability()[0] < 10,
```

Comment:
Can we do `is_hip()` check here to be explicitly disabling it for hip, not everything non-cuda?